File: docs-examples/example-java/src/main/java/io/micronaut/configuration/openapi/docs/resolution/Owner.java
package io.micronaut.configuration.openapi.docs.resolution;

// tag::imports[]
import io.swagger.v3.oas.annotations.media.Schema;
// end::imports[]

// tag::clazz[]
class Owner {

    private Pet cat;
    private Pet dog;

    public Pet getCat() { // <2>
        return cat;
    }

    @Schema(name="MyPet", description="This is my pet") // <3>
    public Pet getDog() {
        return dog;
    }

}
// end::clazz[]

File: docs-examples/example-kotlin/src/main/kotlin/io/micronaut/configuration/openapi/docs/resolution/Owner.kt
package io.micronaut.configuration.openapi.docs.resolution

// tag::imports[]
import io.swagger.v3.oas.annotations.media.Schema
// end::imports[]

// tag::clazz[]
class Owner {

    var cat: Pet? = null // <2>

    @get:Schema(name = "MyPet", description = "This is my pet") // <3>
    var dog: Pet? = null
}
// end::clazz[]
